pkg/db/query/test_queries.go

```diff
@@ -126,7 +126,7 @@ WITH results AS (
            unnest(variants)        AS variant
     FROM prow_test_report_7d_matview
 	WHERE release = @release AND name ~* @testsubstrings
-          AND NOT(@excluded && variants)
+          AND NOT(variants is not null and @excluded && variants)
```

Comment:
when the variants are null, the `&&` operator weirdly seems to come back true. add a check so that null doesn't match exclusions. there are quite a lot of rows with null variants so this might add some results we've not been seeing in the health/overview/install areas. could maybe leave this change out but it helped with devel (not having variants defined for aro-integration) and it seems like a good idea.